composer.json

```diff
@@ -22,7 +22,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.5",
-        "slevomat/coding-standard": "^8.3.0"
+        "slevomat/coding-standard": "^8.3.0",
+        "silverstripe/subsites": "^2.6"
```

Comment:
Ehhhh I'm almost OK with this but it feels a bit wacky to have this in `require-dev`. If possible, could we add it to the pre-install scripts in Github actions?

Reply:
It is a dev requirement :) If someone wants to contribute to the module and build it locally. The tests will failed until they find the github action that adds the module.

But I'm ok with adding it to the build script.